packages/server/test.sh

```diff
@@ -21,6 +21,7 @@ while [[ "$#" -gt 0 ]]; do
   shift
 done
 
+export MUSTER_POSTGRES_CONTROLLER=true
```

Comment:
I thought both @rogelioii and @andersonad  said we don't need both? If so, let's get rid of this environment variable. Since you have already separated out the logic to a separate controller let's just replace the routes outright. That way if we need ES stuff again we can just wire up routes if we need to.

Reply:
It needs to stay as we need to continue using the ES endpoint in Prod due to other work that needs to be done:
- custom fields 
- one off muster requirements
There are tickets for this work in Asana.